charger/openevse.go

```diff
@@ -180,24 +180,27 @@ func (c *OpenEVSE) Status() (api.ChargeStatus, error) {
 func (c *OpenEVSE) Enabled() (bool, error) {
 	ctx, cancel := c.requestContextWithTimeout()
 	defer cancel()
-	overrideResp, err := c.api.GetManualOverrideWithResponse(ctx)
+	overrideResp, err := c.api.GetStatusWithResponse(ctx)
 
 	if err != nil {
 		return false, err
 	}
 
 	if overrideResp.JSON200 != nil && overrideResp.JSON200.State != nil {
-		switch state := *overrideResp.JSON200.State; state {
+		status := *overrideResp.JSON200.Status
+
+		switch status {
 		case "disabled":
 			return false, nil
 		case "enabled", "active":
 			return true, nil
 		default:
-			return false, fmt.Errorf("unknown EVSE state: %s", state)
+			return false, fmt.Errorf("unknown EVSE status: %s", status)
 		}
 	}
 
-	return false, errors.New("invalid EVSE state")
+	// no override:
+	return false, fmt.Errorf("invalid EVSE status")
```

Comment:
Isn't the case here a result != HTTP 200? Then lets return the actual response code to be clearer?

Reply:
I can do that. But I'm also concerned about the "status" property missing in the API response, so just testing for HTTP status code might not suffice.

Reply:
nvm. this is close to before which basically worked. Ok to merge?

Reply:
Yes, please merge.